style/fonts.css

```diff
@@ -22,4 +22,10 @@
   font-family: Gotham Pro Light;
   src: url("./GothamPro/GothamPro-Light.woff2") format("woff2");
   src: url("./GothamPro/GothamPro-Light.woff") format("woff");
-}
+} 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)